src/CLMWrapper.cpp

```diff
@@ -535,24 +539,14 @@ void ClmWrapper::callback(const sensor_msgs::ImageConstPtr& msgIn)
             Vec6d pose_estimate_CLM = CLMTracker::GetCorrectedPoseWorld(clm_models[model], fx, fy, cx, cy);
 
             // Draw it in reddish if uncertain, blueish if certain
-            CLMTracker::DrawBox(disp_image, pose_estimate_CLM, Scalar((1-detection_certainty)*255.0,0,
-                                            detection_certainty*255), thickness, fx, fy, cx, cy);
+            CLMTracker::DrawBox(disp_image, pose_estimate_CLM, Scalar((1-detection_certainty)*255.0,0,detection_certainty*255), thickness, fx, fy, cx, cy);
```

Comment:
long lines are difficult to read: this is why this line was split in two. Please come back to the previous version :+1: